on_device_training/web/README.md

```diff
@@ -0,0 +1,53 @@
+# End-to-end MNIST training demo
+
+This folder contains a demo based on [juharris's project](https://juharris.github.io/train-pytorch-in-js/) that launches a simple model that will train on the [MNIST dataset](http://yann.lecun.com/exdb/mnist/) in your browser.
+
+A live preview of this demo is available [here](https://carzh.github.io/onnxruntime-training-examples/).
```

Comment:
I looked into it but it seems difficult unless I can push directly to ORT training examples repo? Or if somebody has permission to push directly to ORT training examples repo, they can follow the deploy instructions I just added to the README.

The ideal would be a Github Actions flow set up to live deploy to onnxruntime-training-examples.github.io every time the web subdirectory is modified but since running the demo requires multiple steps and file downloads after cloning the repo, it would take some time to figure out how to set up.